tests/test_flow/test_persistence_aip.py

```diff
@@ -29,3 +29,33 @@ def c(a, b):
         return a - b  # 6 (2 * 3) - 5 (2 + 3)
 
     assert builder.build().get("c") == 1
+
+
+@pytest.mark.needs_aip
+def test_aip_not_persisted_dependency(aip_builder):
```

Comment:
I'm a little worried about having too many AIP tests. Each test takes 4-5 minutes when you run with AIP. If it's just this, can we modify the original test to contain a non persisted dep?

Reply:
Good point. I have combined the AIP tests into one.